website/static/js/pages/base-page.js

```diff
@@ -88,6 +88,10 @@ $(document).on('click', '.panel-heading', function(){
     }
 });
 
+$(document).on('click', 'a[href="#"]', function(e) {
```

Comment:
Is there any possibility that this will intercept any of our other click handlers?

Perhaps we shouldn't use `a` tags with `href="#"` and instead use `buttons` with the `btn-link` class.